src/test/java/Safron/NewBalanceUa/Tests/TestSizeBoard.java

```diff
@@ -0,0 +1,33 @@
+package Safron.NewBalanceUa.Tests;
+
+import BasePageAndTestInit.TestInit;
+import Safron.NewBalanceUa.Pages.NewBalanceHeaderViewElements;
+import Safron.NewBalanceUa.Pages.NewBalanceHeaderViewHelper;
+import Safron.NewBalanceUa.Pages.NewBalanceManShoesPageElements;
+import Safron.NewBalanceUa.Pages.NewBalanceManShoesPageHelper;
+import org.testng.annotations.Test;
+
+public class TestSizeBoard extends TestInit {
+
+    @Test
+    public void testManShoesSize() {
+        openUrl("https://newbalance.ua/");
```

Comment:
move openUrl() method on the start of the test (line 20)

Reply:
fix